src/Set.ts

```diff
@@ -31,6 +31,21 @@ export const some = <A>(x: Set<A>, predicate: Predicate<A>): boolean => {
   return found
 }
 
+/**
+  * Projects a Set through a function
+  *  @function */
+export const map = <A, B>(x: Set<A>, bset: Setoid<B>, f: (x: A) => B): Set<B> => {
```

Comment:
Following the convention used so far, shouldn't be

```ts
export const map = <B>(S: Setoid<B>) => <A>(x: Set<A>, f: (x: A) => B): Set<B> => {
  ...
}
```

?

Reply:
ok done